app/commands/v2/unpublish.rb

```diff
@@ -83,6 +83,10 @@ def validate
       end
 
       def unpublish
+        if edition.draft?
+          Edition::Timestamps.live_transition(edition, previous)
```

Comment:
do we have space on the line for this to be `Edition::Timestamps.live_transition(edition, previous) if edition.draft?`

Reply:
Yep just about :)